docs/projects/P5/2_projectfinal.md

```diff
@@ -80,23 +76,30 @@ A brief description of the open source system to which you contributed. You may
 
-3.	**Task Description (per task)** (<0.5 page)    
+3.	**Project Processes and Communication**  (<1 page)   
+Describe the processes and tools the project uses to coordinate among contributors. For example: Are these processes formal or informal? Provide an explicit description (possibly with a diagram) of the acceptance process used for efforts like the task you completed.
+If applicable, include standards or expectations regarding software engineering activities including requirements, architecture, and quality assurance; alternatively mention that no such standards exist.
+
+4.	**Task Description (per task)** (<0.5 page)    
 A description of the tasks you have implemented and a high-level description of how you implemented them.
 
-4.	**Submitted Artifacts (per task)**  
+5.	**Submitted Artifacts (per task)**  
 Evidence of the code, documentation, or other artifacts you produced for the task, and evidence that you submitted them to the project. This would likely be links to publicly available resources (repository, email, pull request, etc).
 
-5.	**QA Strategy** (<1 page)  
+6.	**QA Strategy** (<1 page)  
 Describe which QA activities you performed and justify why you selected these QA activities over others. Describe metrics if appropriate. The justification will likely refer to relevant requirements as well as to the project’s practices. 
 
-6.	**QA Evidence**  
+7.	**QA Evidence**  
 Evidence of your quality assurance activities described above. For example, provide source code or links to source code of tests, provide test protocols, comments or protocols from code reviews, reports from static analysis tools, links to or screenshots from a continuous integration platform, and so forth.
 
-7.	**Plan Updates** (<1 page)  
+8.	**Plan Updates** (<1 page)  
 A description and justification of deviations between your initial plans and your performed activities (as in Homework 2). Changes are expected, but they should be tracked and explained. Describe changes in scope (e.g., fewer tasks) and in the schedule and work allocation. Provide an updated schedule and note differences. Explain the causes of the changes, such as unanticipated risks. 
 
-8.	**Extra Credit**  
-Evidence that your changes have been accepted into the code base of the open source project in forms of links or screenshots. **Note: You can also send the merged PR link via Slack to Michael Hilton by 10th May 1pm to claim this bonus as well**
+9.	**Summary of Final Contributions** (<0.5 page)  
+A table that **briefly summarizes** the contribution(s) each team member made towards completing the task(s).
+
+10.	**Extra Credit**  
+Evidence that your changes have been accepted into the code base of the open source project in forms of links or screenshots. **Note: You can also send the merged PR link via Slack to Rohan Padhye by 18th May 1pm to claim this bonus as well**
```

Comment:
Should this be 18th December? And instead of naming only me, say "the instructors (via Slack group DM or email to both)"